docs/src/pages/quasar-cli/developing-ssr/deploying.md

```diff
@@ -32,3 +32,70 @@ After installing PM2 on your server, your npm start script can look like this in
   "start": "pm2 start index.js"
 }
 ```
+
+## Production scenario
+For serving our Quasar SSR web app in the real world we should consider using Nginx as reverse proxy for leveraging these advantages:
```

Comment:
Technical writing typically avoids using 1st and 2nd person.  For that reason I would consider changing to be something along the lines of:
"Consider using a reverse proxy, such as Nginx, when hosting a Quasar SRR web app in the real world.  The following are just some of the potential benefits:"